src/users/tests.py

```diff
@@ -170,6 +172,47 @@ def test_set_mentor_as_active(self):
         self.mentor.refresh_from_db()
         self.assertEqual(self.mentor.active, True)
 
+    def test_patch_major_correct_and_404(self):
+        
+        #Test correct
+        user_params = {
+            'major' : {
+                'name' : 'Test Major'
+            }
+        }
+
+        resp = self.client.patch(
+            self.mentors_update_url,
+            data=user_params,
+            format='json',
+        )
+
+        self.mentor.refresh_from_db()
+        self.assertEqual(self.mentor.major.name, 'Test Major')
+        self.assertEqual(len(Major.objects.all()), 1) 
+        self.mentor2 = factories.MentorFactory()
+        
+
+        #Test Error
+        user_params = {
+            'major' : {
+                'name' : 'Wrong'
+            }
+        }
+
+        resp = self.client.patch(
+            self.mentors_update_url,
+            data=user_params,
+            format='json',
+        )
+    
+        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+        self.mentor.refresh_from_db()
+        self.assertEqual(self.mentor.major.name, 'Test Major')
+        self.assertEqual(len(Major.objects.all()), 1) 
```

Comment:
for test cases, typically you want to test exactly 1 behavior per tests.  in this case, i would split up the #Test correct and #Test Error into two test cases.

Reply:
kk this looks good now.